internal/install/application_configuration_test.go

```diff
@@ -14,43 +14,43 @@ import (
 func TestSelectElasticAgentImageName_NoVersion(t *testing.T) {
 	var version string
 	selected := selectElasticAgentImageName(version, "")
-	assert.Equal(t, selected, elasticAgentLegacyImageName)
+	assert.Equal(t, elasticAgentWolfiImageName, selected)
```

Comment:
This is the test changing the value, it would return the wolfi image instead.

Reply:
In principle it LGTM. Maybe another option is to return an error if the version is empty? Not sure about the implications of the change, but would look safer to me to know where/if we are passing an empty version.

Reply:
Commands like `elastic-package stack down` do not set any version, and if a error is returned here, it will fail the command.